syntax = "proto3";

package base;

service BaseService {
  rpc SayHello (HelloRequest) returns (HelloReply);
  rpc SayGoodBye(GoodByeRequest) returns (ByeReply);
}

message HelloRequest {
  string name = 1;
}

message HelloReply {
  string message = 1;
}

message GoodByeRequest {
  string name = 1;
}

message ByeReply {
  string message = 1;
}
